doc/developer_guide/HNS_error_bestiary.md

```diff
@@ -78,11 +78,15 @@ The following chapter contains a list of HNS errors encountered throughout devel
 
 ### Natural habitat:
 
-TODO
+1.  When attempting to create a transparent HNS network
+    * when creation of another network or VMSwitch is already in progress
+      (eg. we've just started Docker service and it tries to create the NAT network).
+
+    We can work around this bug by retrying after a few seconds.
 
 ### Reproduction:
 
-TODO
+Try to create multiple transparent HNS networks in a loop simultaneously with multiple processes.
```

Comment:
Wouldn't it create only a single vSwitch?

Reply:
Yep, you're right. I think high load of creating any HNS networks could trigger this bug.

Reply:
@krdln this should be included in the doc then :)

Reply:
highload thing pls